CMake/SlicerCPack.cmake

```diff
@@ -502,15 +502,21 @@ if(CPACK_GENERATOR STREQUAL "NSIS")
   set(CPACK_NSIS_FINISH_TITLE "Completed the ${PACKAGE_APPLICATION_NAME} Setup Wizard")
   set(CPACK_NSIS_FINISH_TITLE_3LINES True)
 
+  # -------------------------------------------------------------------------
+  # Graphics Preference
+  # -------------------------------------------------------------------------
+  set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS
+"WriteRegStr SHCTX \\\"SOFTWARE\\\\Microsoft\\\\DirectX\\\\UserGpuPreferences\\\" \\\"$INSTDIR\\\\bin\\\\${APPLICATION_NAME}App-real.exe\\\" \\\"GpuPreference=2;\\\"
```

Comment:
Is GpuPreference set to 2 always the GPU ? I am wondering if we could include a link to the relevant documentation of any.

Reply:
I have added more code comments describing the available options and that "2" specifically maps to the "High performance" selection. "1" maps to the "Power saving" selection which is generally the integrated graphics and "0" is the default "Let Windows decide" where Windows has been choosing the integrated graphics for Slicer.

Unfortunately I have not found an official Microsoft documentation page describing this graphics preference setting. There are many other third party websites, but none worth memorializing as a link in the code.

Reply:
In some cases as mentioned in the original post, systems with only integrated graphics will have the same option displayed for the "Power saving" and "High performance" option.

<img width="525" height="197" alt="image" src="https://github.com/user-attachments/assets/70afd986-4bdc-4f95-9273-048a6f5b3816" />